apps/prairielearn/src/tests/comments.test.ts

```diff
@@ -0,0 +1,37 @@
+import { assert } from 'chai';
+
+import { isRenderableComment } from '../lib/comments.js';
+
+describe('isRenderable should determine non-empty comments', function () {
```

Comment:
Nit, conventionally for unit tests, we use the function name as the name of the describe block.

```suggestion
describe('isRenderableComment', function () {
```

Reply:
Updated locally in [54216f0](https://github.com/PrairieLearn/PrairieLearn/pull/11947/commits/54216f0435ad1be2f3a449525f33fa75076bccdf).